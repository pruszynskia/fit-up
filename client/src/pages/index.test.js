import React from 'react';
import renderer from 'react-test-renderer';

import App from '../pages/index';

jest.mock('../pages/index', () => 'App')

describe("<App /> with Enzyme", () => {
    it('Should render App without crushing', async() => {

      const tree = renderer.create(<App />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });