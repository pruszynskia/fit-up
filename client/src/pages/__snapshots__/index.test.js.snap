// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> with Enzyme Should render App without crushing 1`] = `<App />`;
